prov/efa/src/rxr/rxr_ep.c

```diff
@@ -1634,6 +1650,28 @@ void rxr_ep_progress_internal(struct rxr_ep *ep)
 		dlist_remove(&read_entry->pending_entry);
 	}
 
+	/*
+	 * Send local read requests until finish or error encoutered
+	 */
+	dlist_foreach_container_safe(&ep->local_read_pending_list, struct rxr_local_read_entry,
+				     local_read_entry, pending_entry, tmp) {
+		/*
+		 * The core's TX queue is full so we can't do any
```

Comment:
There is no core so "The TX queue is full" is fine here.

Reply:
Thanks! fixed